projects/step-core/src/lib/modules/dynamic-forms/components/expression-input/expression-input.component.html

```diff
@@ -1,34 +1,30 @@
-<mat-form-field floatLabel="always" class="step-view" [class.ng-invalid]="!!_ngControl.invalid || isParentInvalid">
-  <mat-label *ngIf="label">{{ label }}</mat-label>
+<step-form-field [class.ng-invalid]="!!_ngControl.invalid || isParentInvalid" [class.with-tooltip]="tooltip">
+  <step-label [attr.show-required-marker]="showRequiredAsterisk || null" *ngIf="label">{{ label }}</step-label>
```

Comment:
why do we need the ```|| null``` part

Reply:
Without `null` attribute will remain, even if `showRequiredAsterisk` will be `false`.
In case of `null` angular will remove the attribute

Reply:
nice catch